app-src/set-up-parent-frame-communications.js

```diff
@@ -21,7 +22,11 @@ function setUpParentFrameCommunications(document) {
 }
 
 function playVideos() {
-  paella.player.videoContainer.play();
+  $(document).trigger(paella.events.play);
```

Comment:
Oh. I think he (or maybe you) mentioned this specifically as a limitation in his implementation, so I bet @karendolan's solution here is the correct one.